modules/3d/include/opencv2/3d.hpp

```diff
@@ -2603,6 +2603,44 @@ of image, we can notice that on image a) these points are distorted.
 CV_EXPORTS_W void undistortImage(InputArray distorted, OutputArray undistorted,
     InputArray K, InputArray D, InputArray Knew = cv::noArray(), const Size& new_size = Size());
 
+/**
+@brief Finds an object pose from 3D-2D point correspondences for fisheye camera moodel.
+@param objectPoints Array of object points in the object coordinate space, Nx3 1-channel or
```

Comment:
Keep empty line after `@brief`.

Removal is unnecessary change against 4.x:

https://github.com/opencv/opencv/blob/12aa0fe89819c02db53677c5749771a1007f623e/modules/calib3d/include/opencv2/calib3d.hpp#L4059-L4061